src/components/atoms/StyledNewTweetButton.tsx

```diff
@@ -0,0 +1,25 @@
+"use client";
+
+import styled from "@emotion/styled";
+
+
+const NewTweetButton = styled.button`
+  background-color: lightblue;
+  width: 80px;
+  height: 50px;
+  position: fixed;
+  z-index: 1000;
```

Comment:
niga stop adding this absurd z-indeces.

why on earth should a z-index value be more than 10 ???